incuna-sass/components/_typography.sass

```diff
@@ -0,0 +1,216 @@
+// Based on Simple Typography by Adam Marsden and contributors. 
+// Licensed under the MIT license.
+// https://github.com/AdamMarsden/simple-typography
+
+// Font Family
+$base-font-family: 'Lato' !default
```

Comment:
oh yeah, I don't think I meant to leave this in here, I was just going to put `sans-serif` I think